internal/cmd/iam/command_rbac_role_describe.go

```diff
@@ -51,6 +53,12 @@ func (c *roleCommand) ccloudDescribe(cmd *cobra.Command, role string) error {
 		streamCatalogNamespace.Value(),
 	}
 
+	// check if IdentityAdmin is enabled
+	ldClient := v1.GetCcloudLaunchDarklyClient(c.Context.PlatformName)
+	if featureflags.Manager.BoolVariation("auth.rbac.identity_admin.enable", c.Context, ldClient, true, false) {
+		namespacesList = append(namespacesList, identityNamespace.Value())
```

Comment:
Would it be possible to create an API that returns the list of namespaces, so we don't have to keep making trivial PRs like this one?

Reply:
We use new namespaces to "hide" new roles until they are ready for GA, or if we intend to use it for internal roles only. You can see all the actual namespaces here: https://github.com/confluentinc/ce-kafka/blob/master/ce-rbac/src/main/java/io/confluent/security/rbac/Role.java

Specifically for this role it is planned on being moved to the `public` namespace as a post-GA task so we would just have to delete these lines and update tests.